.github/prompts/error-consolidation.md

```diff
@@ -1,46 +1,60 @@
 
 - Add the contents of the source file to the destination.md file.
-  - Include the source error code in the YML header for `f1_keywords` and `helpviewer_keywords`.
+  - Include the source error code in the YML header for f1_keywords and helpviewer_keywords.
   - Add an entry with an anchor for the error error code and its corresponding error message.
   - Add the contents of the source file as a new H2 in the destination file.
-- Add a redirection for the source file in the file `.openpublishing.redirection.csharp.json`. Make the destination point to destination file. Place the new entry in sorted order based on 'source_path_from_root'.
+  - Add a redirection for the source file in the file .openpublishing.redirection.csharp.json. Make the destination point to destination file. Place the new entry in sorted order based on 'source_path_from_root'.
 - Update the TOC file:
   - Add the error code to the list of display names in the TOC for the destination file, sorted by numeric error code.
   - Remove the TOC entry for the source file.
-- Finally, delete the source markdown file. 
+  - Finally, delete the source markdown file.
 
 ## Search for other related articles that may be missed.
 
-Search all files in the docs/csharp/language-reference/compiler-messages and the docs/csharp/misc folder for any other errors and warnings that involve string literals or constants. Give me a list to review for possible additional consolidation.
+Search all files in the docs/csharp/language-reference/compiler-messages and the docs/csharp/misc folder for any other errors and warnings that involve operator overloading. Give me a list to review for possible additional consolidation. Don't make any edits until I approve.
 
 ## Final search in roslyn source
 
+Let's check undocumented errors and the roslyn source for any missing errors.  For every error code listed in "sorry-we-don-t-have-specifics-on-this-csharp-error.md" under the `f1_keywords` front matter, do the following:
+1. Find that number as a constant in `ErrorCodes.cs`.
+2. Locate the corresponding `data` element in CSharpResources.resx. The `name` atttribute should match the number of the constant.
+3. Read the error message found in the `<value>` element that is a child of that `<data>` element.
+Give me a list of all error numbers and corresponding error messages that relate to operator overloading.
+
 To make sure you've found all related errors, we'll check the source.  Look in `CSharpResources.resx` for any elements where the `<value>` element is a message related to preprocessor tokens. The symbolic constant for that value is in the `name` attribute on the parent `data` element. Find that value in `ErrorCodes.cs`. It will map to the compiler error code, where the code is "CS" followed by the number as a four digit number. Build a list of any related errors, but don't make any edits yet.
 
-For each new file:
+I'll give you error codes one by one. For each, I want you to do the following:
```

Comment:
Where is this prompt used? Is it a local thing that you invoke, or from the agent panel in GitHub?